### DISCUSSION TITLE
Grid view: name on 2 lines

### DISCUSSION THREAD

**Discussion by Geobert** [TeamAmaze/AmazeFileManager#2875] [Ideas]
Hi,

I don't know if it's possible but I couldn't find any option on that.

I'd like to have the file/folder name on 2 lines instead of having the scroll (and get rid of the date on the 2nd line).
If the name is too long, put `...` at the end :)

**Comment by VishnuSanal** [comment]
Hello!

> I'd like to have the file/folder name on 2 lines instead of having the scroll (and get rid of the date on the 2nd line).

Turning off "Show last modified time and date", "Show sizes" and "Enable marquee effect for file names" from the settings will give this behaviour.

> If the name is too long, put `...` at the end :)

The `...` will be on the middle. I don't think there's an option to move this to the end.

**Reply by Geobert** [reply]
Thank you for your answer! It is almost what I'm looking for indeed! 

The missing part it spanning the name on 2 lines (I'm talking about grid view)

**Reply by VishnuSanal** [reply]
> The missing part it spanning the name on 2 lines (I'm talking about grid view)

@VishalNehra What do you think? Should this be an issue? I personally think that that won't look good.

**Reply by Geobert** [reply]
I have this on https://play.google.com/store/apps/details?id=com.speedsoftware.explorer&hl=en_US&gl=US

and it looks quite fine to me :) But we're entering personal taste territory, so no good answer here :D
